src/data/lovelace.ts

```diff
@@ -21,6 +21,13 @@ export interface LovelaceCardConfig {
   [key: string]: any;
 }
 
+export interface ActionConfig {
```

Comment:
This is the wrong approach. We should have a separate interface for each action and then have `ActionConfig` be a union interface: `ActionConfig = CallServiceActionConfig & MoreInfoActionConfig`